.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,150 @@
+name: "P4 Control Plane CI Pipeline"
+
+on:
+  push:
+    branches:
+      - main
+      - ipdk_v*
+  pull_request:
+    branches:
+      - main
+      - ipdk_v*
+
+concurrency:
+  # If workflow for PR or push is already running, stop it and start a new one.
+  group: build-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions: read-all
+
+env:
+  #---------------------------------------------------------------------
+  # Environment variables
+  #---------------------------------------------------------------------
+  SDE_REPOSITORY: ipdk-io/p4dev.dpdk-sde
+  SDE_TAG: 2023.10.1
```

Comment:
Where does this tag come from?

Reply:
The answer to your question may be found here:

- [dpdk-sde 2023.10.1](https://github.com/ipdk-io/p4dev.dpdk-sde/releases/tag/2023.10.1)

I've scheduled a CI Pipeline Review (Teams meeting) for 9:00 am Wednesday to discuss the mechanics of the revised pipeline. I figured a guided tour would be easier than a written explanation.